### PR TITLE
Feat/videoplayer inverse

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

#2212

Adds the inverse theme for Videoplayer

### CODE REVIEW

File: packages/react/src/components/VideoPlayer/VideoPlayer.js [hunk 1]
```diff
@@ -17,10 +18,13 @@ const { prefix } = settings;
 /**
  * VideoPlayer component
  *
- * @param {string} videoId Kaltura video id
+ * @param {object} props props object
+ * @param {string} props.type theming options
```
Comment (jeffchew) [inline packages/react/src/components/VideoPlayer/VideoPlayer.js:22]: General question, is `type` the right prop name for this or we want to rename to something else?

cc: @andysherman2121 @kennylam @reesedownes @annawen1 @RaphaelAmadeu @guilhermelMoraes

Comment (annawen1) [reply packages/react/src/components/VideoPlayer/VideoPlayer.js:22]: should we just have it as a boolean called `inverse`?

File: packages/styles/scss/components/video-player/_video-player.scss [hunk 2]
```diff
@@ -38,6 +38,12 @@
       }
     }
   }
+  .#{$prefix}--video-player--inverse {
+    background-color: $inverse-02;
+  }
+  .#{$prefix}--video-player__video-description--inverse {
+    color: #ffffff;
```
Comment (annawen1) [inline packages/styles/scss/components/video-player/_video-player.scss:45]: can we use a token for this?

Comment (andysherman2121) [reply packages/styles/scss/components/video-player/_video-player.scss]: updated with token

File: packages/react/src/components/VideoPlayer/VideoPlayer.js (hunk 1 above)
Comment (andysherman2121) [reply packages/react/src/components/VideoPlayer/VideoPlayer.js]: Updated to `inverse`

File: packages/react/src/components/VideoPlayer/VideoPlayer.js [hunk 3]
```diff
@@ -32,10 +41,23 @@ const VideoPlayer = ({ videoId, showDescription }) => {
...
+    `${prefix}--video-player`,
+    { [`${prefix}--video-player--inverse`]: inverse },
+    customClassName
+  );
+  const textclass = cx(
+    `${prefix}--video-player__video-description`,
+    {
+      [`${prefix}--video-player__video-description--inverse`]: inverse,
+    },
+    customClassName
```
Comment (annawen1) [inline packages/react/src/components/VideoPlayer/VideoPlayer.js:54]: I don't think we need to add inverse modifier for the text. We can just use the 

`.#{$prefix}--video-player--inverse .#${prefix}--video-player__video-description {}` selector instead in the css

Comment (andysherman2121) [reply packages/react/src/components/VideoPlayer/VideoPlayer.js:54]: updating

**Pull request merged**